packages/core/src/App/Components/Layout/Header/menu-links.jsx

```diff
@@ -51,7 +56,12 @@ const CashierTab = observer(() => {
     const toggleModal = () => {
         if (toggle_modal_routes && !has_any_real_account) {
             toggleReadyToDepositModal();
-        } else if (window.location.pathname === routes.traders_hub) {
+        } else if (
+            window.location.pathname === routes.traders_hub &&
+            has_any_real_account &&
+            !is_virtual &&
```

Comment:
just a question: do we need !is_virtual check here?
I remember content flag already set based on real and virtual. If virtual , content flag will be **CR_DEMO**

Reply:
just an extra check